app-office/tuxcards/tuxcards-2.2.1.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="The Hierarchical Notebook"
+DESCRIPTION="An Hierarchical notebook to enter and manage every kind of notes \
+and ideas in a structured manner.
+It was created out of the desire to let my own chaos of papers, \
+notes, and post-it's vanish.
+Those items are useful but the chaos I produced was not.
+With TuxCards you have a tool at your hand to free your mind by creating \
+notes using richtext and images.
+It has been proven to work on Haiku, Linux, Mac and Windows."
+HOMEPAGE="https://www.tuxcards.de/"
+COPYRIGHT="2015-2018 Alexander Theel"
+LICENSE="GNU GPL v2"
+srcGitrev="f496695228c2fec4942597c74abde746076eac09"
+REVISION="1"
+SOURCE_URI="https://github.com/eli-schwartz/tuxcards/archive/$srcGitrev.tar.gz"
+CHECKSUM_SHA256="ea7af79b974608c769baf0ce1e3d425e0a9784e651d809c38e2c33933836bd14"
+SOURCE_DIR="tuxcards-$srcGitrev"
+ADDITIONAL_FILES="tuxcards.rdef.in"
```

Comment:
Some cosmetics:
```diff
-srcGitrev="f496695228c2fec4942597c74abde746076eac09"
-REVISION="1"
-SOURCE_URI="https://github.com/eli-schwartz/tuxcards/archive/$srcGitrev.tar.gz"
-CHECKSUM_SHA256="ea7af79b974608c769baf0ce1e3d425e0a9784e651d809c38e2c33933836bd14"
-SOURCE_DIR="tuxcards-$srcGitrev"
-ADDITIONAL_FILES="tuxcards.rdef.in"
+REVISION="1"
+srcGitRev="f496695228c2fec4942597c74abde746076eac09"
+SOURCE_URI="https://github.com/eli-schwartz/tuxcards/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="ea7af79b974608c769baf0ce1e3d425e0a9784e651d809c38e2c33933836bd14"
+SOURCE_FILENAME="tuxcards-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="tuxcards-$srcGitrev"
+ADDITIONAL_FILES="tuxcards.rdef.in"
```

Reply:
We use "srcGitRev" instead of "srcGitrev" (minor change)
srcGitRev comes after REVISION and before SOURCE_URI
Added SOURCE_FILENAME

Reply:
ok done.. thanks @Begasus